src/conf_mode/containers.py

```diff
@@ -215,20 +215,32 @@ def apply(container):
                 # Currently the best way to run a command and immediately print stdout
                 print(os.system(f'podman pull {image}'))
 
+                # Bind volume
+                volume = ''
+                if 'bind_volume' in container['name'][name]:
+                    volume = '-v '
+                    volume += " -v ".join(f"{v}" for v in container['name'][name]['bind_volume'])
+
                 # Check/set environment options "-e foo=bar"
                 env_opt = ''
                 if 'environment' in container_config:
                     env_opt = '-e '
                     env_opt += " -e ".join(f"{k}={v['value']}" for k, v in container_config['environment'].items())
 
+                # Publish ports
+                publish_port = ''
+                if 'publish_port' in container['name'][name]:
```

Comment:
There is already container_config config in place which has the same content as container['name'][name]. Please adjust the syntax.

Reply:
done